test/BaseTest.t.sol

```diff
@@ -10,7 +10,7 @@ import { PushCoreV3 } from "contracts/PushCore/PushCoreV3.sol";
 import { PushCoreMock } from "contracts/mocks/PushCoreMock.sol";
 import { EPNSCoreProxy, ITransparentUpgradeableProxy } from "contracts/PushCore/EPNSCoreProxy.sol";
 import { EPNSCoreAdmin } from "contracts/PushCore/EPNSCoreAdmin.sol";
-import { PushCommV3 } from "contracts/PushComm/PushCommV3.sol";
```

Comment:
Does this mean PUSHCommV3 is not included in BaseTest at all?

PushCommV3 and PushCommETHV3 both contract will eventually have different sets of functions that will require testing. Any reason to remove PushCommV3 completely ?

Reply:
Sure, but since we are arranging the test files as of now, better to add PushCommV3's initializers in the BaseTest too so it can be used when need be. @0xNilesh

Reply:
LGTM  . merging this PR